packages/io/src/io-util.ts

```diff
@@ -166,3 +166,8 @@ function isUnixExecutable(stats: fs.Stats): boolean {
     ((stats.mode & 64) > 0 && stats.uid === process.getuid())
   )
 }
+
+// Get the path of cmd.exe in windows
+export function getCmdPath(): string {
+  return process.env['COMSPEC'] ?? `C:\\WINDOWS\\system32\\cmd.exe`
```

Comment:
we might want to fallback to `'cmd.exe'` instead of `C:\Windows\System32\cmd.exe`

Reply:
Since we're checking explicitly for quotes already, we should be safe to fallback to simply `cmd.exe`